data-raw/bucharest.R

```diff
@@ -0,0 +1,108 @@
+library(dplyr)
+library(osmdata)
+library(rlang)
+library(sf)
+library(usethis)
+
+
+get_osm_bb <- function(city_name) {
+  bb <- getbb(city_name)
```

Comment:
```suggestion
  bb <- osmdata::getbb(city_name)
```

Reply:
Is this change really needed, as osmdata is imported as a library above?

Reply:
If this and similar changes are made, should we also remove importing osmdata above (`library(osmdata)`)?

Reply:
For me, locally lintr does not give any linting issues without these changes. I will check what is different compared to the lintr run in GitHub.

Reply:
> If this and similar changes are made, should we also remove importing osmdata above (`library(osmdata)`)?

You are right. I guess that is happening because dependencies are managed with `environment.yml` which is not the standard way in which an R package manages dependencies, afik. I would start resolving this by adding osmdata to `Suggests` in `DESCRIPTION`.

Reply:
It seems like that has resolved the lintr and part of r-CMD-check issues without needing to call the osmdata functions explicitly.

Remaining problems in r-CMD-check are with undocumented code objects and data sets.

Reply:
`usethis::use_build_ignore("data-raw")` should fix the note on `data-raw`. I guess the same can be done with `environment.yml`

Reply:
`devtools::document()`, also run locally should document the new data object as well